drivers/storage/portworx/component/csi.go

```diff
@@ -114,12 +114,13 @@ func (c *csi) Reconcile(cluster *corev1alpha1.StorageCluster) error {
 
 func (c *csi) Delete(cluster *corev1alpha1.StorageCluster) error {
 	ownerRef := metav1.NewControllerRef(cluster, pxutil.StorageClusterKind())
-	// We don't delete the service account for CSI because it is part of CSV. If
-	// we disable CSI then the CSV upgrades would fail as requirements are not met.
-	if err := k8sutil.DeleteClusterRole(c.k8sClient, CSIClusterRoleName, *ownerRef); err != nil {
+	if err := k8sutil.DeleteServiceAccount(c.k8sClient, CSIServiceAccountName, cluster.Namespace, *ownerRef); err != nil {
```

Comment:
Does the older comment not hold true anymore?

Reply:
In 1.3 we removed the service accounts (lighthouse, pvccontroller, csi) from the operator CSV as it was needed there and started handling deletion of SA ourselves. We missed SA for csi.